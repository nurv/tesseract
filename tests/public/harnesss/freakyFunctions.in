int foo(int a) { return a; }
foo(1)
String foo(int a, String b){ return b+a; }
foo(1,"b")
int fact(int v) {if(v == 0){return 1;}else{return v*fact(v-1);}}
fact(5)
int fact(int v){if(v == 0)return 1;else return v*fact(v-1);}
fact(5)
int fact(int v){{if(v == 0)return 1;else return v*fact(v-1);}}
fact(5)
int fact(int v){{System.out.print("garth knight");}{if(v == 0)return 1;else return v*fact(v-1);}}
fact(5)
int fact(int v){(true)?v=v-1:v=v+1;{if(v == 0)return 1;else return v*fact(v-1);}}
fact(5)
int fact(int v){(true&&true||false&&true)?v=v-1:v=v+1;{if(v == 0)return 1;else return v*fact(v-1);}}
fact(5)
String foo="blah";
