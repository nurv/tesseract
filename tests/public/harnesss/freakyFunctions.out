> > 1
> > "b1"
> > 120
> > 120
> > 120
> > garth knightgarth knightgarth knightgarth knightgarth knightgarth knight120
> > 8
> > 8
> > 